src/lib/stitching/vortex/schema.ts

```diff
@@ -24,9 +25,14 @@ export const executableVortexSchema = ({
   return transformSchema(schema, [
     // we don't want pricingContext to be a root query field, it is
     // accessible through artwork
```

Comment:
I’m a little confused about where this stitching occurs, could you link me to the file?

Reply:
It doesn't yet, it should be in `src/lib/stitching/vortex/stitching.ts` when it does

Reply:
Ah kk, thanks 👍

Reply:
oooh that looks like what I should have done in #1618 rather than directly adding to the `Artwork` type 🤔 I wonder if it's worth going back and changing it?

Reply:
Yep, it is worth it, all the vortex logic should live in those three files (link/schema/stitch)

Reply:
Yeah absolutely, please–that is not stitching, it’s a gateway and we shouldn’t be doing that any more.

Reply:
This is done now. Thanks to @orta for pairing :bow: